src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -1,59 +1,139 @@
 package codeu.controller;
 
 import codeu.model.data.Conversation;
+import codeu.model.data.Message;
 import codeu.model.data.User;
 import codeu.model.store.basic.ConversationStore;
 import codeu.model.store.basic.MessageStore;
 import codeu.model.store.basic.UserStore;
+
+
 import java.io.IOException;
 import java.time.Instant;
-import java.util.List;
-import java.util.UUID;
+import java.util.*;
```

Comment:
nit: import each class separately unless you're using more than 50% of util library. This is more important for bigger projects, as importing the whole library bloats your binary size and may also cause unnecessary class name conflicts

Reply:
Done